### ISSUE TITLE
Show Thumbnails/Preview for Uploaded Videos

### ISSUE THREAD

**Issue by emoxam** [RocketChat/Rocket.Chat#32009]
### Description:

No Thumbnails/Preview for Uploaded Videos

### Steps to reproduce:

1. send avi file fox example
2. see there is no Thumbnails/Preview for Uploaded Videos

### Expected behavior:

Preview/thumbnail with ability to play.

### Actual behavior:

Is it fixed ? Have the the same issue on the 6.6.2 Snap install at ubuntu 20.04.3 LTS.
Web page and windows app shows this

![image](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/3593601/b2b30805-8af7-45a7-8067-2c3f0b58ff01)

### Server Setup Information:

- Version of Rocket.Chat Server: 6.6.2
- Operating System: 
- Deployment Method: snap
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version: both
- Operating System: Windows 11

**Comment by athul-22** [comment]
Shall I start work on this feature

**Comment by lovekyleman** [comment]
Rocketchat iOS app needs thumbnails for uploaded videos

**Comment by satvik2131** [comment]
Hi @emoxam , I guess there is some issue with the file itself , I have tried and able to see a thumbnail, please correct me if I got something wrong .
![Uploading Screenshot from 2024-03-18 20-37-31.png…]()

**Comment by emoxam** [comment]
@satvik2131 is it avi file? Can your share a screenshot?

**Comment by satvik2131** [comment]
Yes , you are right avi files are not showing thumbnails , is this reason is true , please correct if it is wrong ?
![Screenshot from 2024-03-21 23-09-58](https://github.com/RocketChat/Rocket.Chat/assets/59243339/3024ad71-efdf-4e85-958b-383c7523da38)

**Comment by emoxam** [comment]
> Yes @emoxam , you are right avi files are not showing thumbnails , is this reason is true , please correct if it is wrong ? ![Screenshot from 2024-03-21 23-09-58](https://private-user-images.githubusercontent.com/59243339/315565350-3024ad71-efdf-4e85-958b-383c7523da38.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwNDU2NTksIm5iZiI6MTcxMTA0NTM1OSwicGF0aCI6Ii81OTI0MzMzOS8zMTU1NjUzNTAtMzAyNGFkNzEtZWZkZi00ZTg1LTk1OGItMzgzYzc1MjNkYTM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIxVDE4MjIzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1ZDAxNDY5MTU4OTA0Y2MzMjliYzdiOTBhZGY1MjlmOGE2ZTI1ZWM4ZDZhNmY3MjFlZDdjZjcyYzllM2ZkYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.XPpoqwNVi1ZcSM_xMoZ1zLb7HnTsqlfIYYJGXrRgkoY)

Maybe, i am not a developer. Can it be played without downloading?

**Comment by satvik2131** [comment]
Hi @emoxam , actually the issue is by default avi does not provide preview , but for that case we can convert it to other format and have a preview and it's a feature actually . So , I need guidance of any maintainer to implement this .
